plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -682,6 +721,17 @@ fu_plugin_coldplug (FuPlugin *plugin, GError **error)
 		}
 	}
 
+	/* firmware may lie */
+	if (!fu_plugin_get_config_value_boolean(plugin, "DisableCapsuleUpdateOnDisk")) {
+		g_autoptr(GError) error_cod = NULL;
+		if (!fu_backend_uefi_check_cod_support(&error_cod)) {
+			g_debug("failed to check CapsuleOnDisk support: %s", error_cod->message);
```

Comment:
nit: In the scenario it's not supported, this message will emit `failed to check CapsuleOnDisk support: Capsule-on-Disk is not supported` which seems wrong.
It should be instead `not using CapsuleOnDisk support: %s`

Reply:
Fixed, ta.